requirements.txt

```diff
@@ -1,28 +1,79 @@
+amqp==2.1.4
+anymarkup==0.7.0
+anymarkup-core==0.7.1
 appdirs==1.4.3
+argh==0.26.2
 atomicwrites==1.2.1
 attrs==18.2.0
 beautifulsoup4==4.6.3
+billiard==3.5.0.4
+boto==2.49.0
+boto3==1.9.44
+botocore==1.12.44
 bs4==0.0.1
+celery==4.2.1
 certifi==2018.11.29
 chardet==3.0.4
+Click==7.0
+codegen==1.0
+colorama==0.3.9
+configobj==5.0.6
+coverage==4.5.2
 cssselect==1.0.3
+docutils==0.14
 fake-useragent==0.1.11
+flake8==3.6.0
+flake8-polyfill==1.0.2
+git2json==0.2.3
+gitdb2==2.0.5
+GitPython==2.1.11
+graphviz==0.10.1
 idna==2.7
+jmespath==0.9.3
+jsl==0.2.4
+json5==0.6.1
+jsonschema==2.6.0
+kombu==4.2.1
+logutils==0.3.5
 lxml==4.2.5
+mando==0.6.4
+mccabe==0.6.1
 more-itertools==4.3.0
 parse==1.9.0
+pathtools==0.1.2
 pluggy==0.8.0
+psycopg2==2.7.6.1
 py==1.7.0
 pycodestyle==2.4.0
 pyee==5.0.0
+pyflakes==2.0.0
 pyppeteer==0.0.25
 pyquery==1.4.0
 pytest==4.0.1
```

Comment:
I think we shouldn't mix runtime and test dependencies here. I'd move test dependencies to `test-requirements.txt` under `tests/` directory.

Reply:
Do we really need `test-requirements.txt` file? For example `pyflakes` are installed in the `detect-common-errors.sh` script itself.